src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/.template.config/template.json

```diff
@@ -434,7 +434,15 @@
     },
     "RequiresHttps": {
       "type": "computed",
-      "value": "(OrganizationalAuth || IndividualB2C || !NoHttps)"
+      "value": "(OrganizationalAuth || IndividualB2CAuth)"
```

Comment:
I'm missing something. Where is `IndividualB2C` being renamed❔ Looks like there are two other template.json files w/ similar changes below

Reply:
Right, this was the confusion before. The **value** of the `auth` property is "IndividualB2C" but the computed property to check is actually `IndividualB2CAuth`.